### ISSUE TITLE
Feature request: Read GITHUB_REPO and GITHUB_USER from `.git/config`

### ISSUE THREAD

**Issue by ivotron** [aktau/github-release#30]
Is it OK to assume that this can be inferred from the `[origin]` entry in `.git/config`?

**Comment by aktau** [comment]
Yes, this is definitely feasible if a user is within a git repository. One would need to scan upwards for a git repo and then parse the git config format. PRs welcome if you like :).

**Comment by Tenzer** [comment]
Alternatively `git config` or an API version of it can help detect and parse a config file for a Git repository.

**Comment by jaymecd** [comment]
would be awesome. 
👍

**Comment by drewkerrigan** [comment]
Hello, before noticing this issue, I hacked together #49 which I think addresses the request from this issue as well. Comments and critiques welcome.
